modules/ROOT/pages/osv2-usage.adoc

```diff
@@ -8,17 +8,40 @@ endif::[]
 
 You can view a chart indicating how many API requests have been consumed.
 
-* Every API call (via the connector or API) to Object Store v2 counts as one API request.
+* Only events count towards your monthly quota.
 +
-All requests to the broker API, such as creating and deleting objects, are counted against the quota.
-* Up to 100 KB of object-payload data can be transferred in a single API call,
-which bills as one request.
-* Each additional 100 KB of payload data is charged as an additional API request. 
-* Monthly billing usage information is delayed by 24 hours.
+Each event (such as create, modify, or delete) on a store or an item counts as one billable unit against the quota.
++
+See <<billevents>>.
+* For each billable unit, up to 100 KB of object-payload data counts as one request against the quota.
++
+Each additional 100 KB of payload data is charged as an additional request. 
 
+Monthly billing usage information is delayed by 24 hours.
 //Shared Usage Overage 
 include::partial$osv2-faq.adoc[tag=faqUsageOverage]
 
+[[billevents]]
+.Billable Unit Events
+[%header,cols="15,15,60a"]
+|===
+|Type|Event|Description  
+.6+|Store|`CREATE_STORE` | Create an object store.
+| `DELETE_STORE` | Delete an object store.
+| `DELETE_ALL_STORES` | Delete all object stores.
+| `MODIFY_STORE` | Modify an object store.
+| `CLEAR_STORE` | Removes the contents of an object store.
+| `RETRIEVE_STORE` | Retrieve the contents of an object store.
+.8+|Item| `CLEAR_ITEM` | Clear a key.
+| `DELETE_ITEM` | Delete a key or value.
+|`CREATE_ITEM` | Create a key or value.
+|`MODIFY_ITEM` | Modify a key or value.
+|`NACK_ITEM` | Provide negative acknowledgment of a key or value?
```

Comment:
This is for keys that are inserted using confirmable == true (https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/object-store-v2/minor/1.0/console/method/%23360/). 

This is used for idempotent filter. 

The key can be acked / or nacked within the confirmation ttl.